uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
```

Comment:
Step 0: Install the version of Dev Home you want to test. I think it's worth pointing out specially for the dev build case, as one would probably expect it to test with any changes made to the code without thinking to install it.